nginx.yaml

```diff
@@ -132,3 +134,15 @@ subpackages:
                 install -m644 -D stream.conf ${{targets.subpkgdir}}/etc/nginx/stream.d/
               ;;
             esac
+    - name: nginx-package-config
+      description: Config supplied in packaged nginx.org bundles at https://hg.nginx.org/pkg-oss/
+      pipeline:
+        - runs: |
+            mkdir -p ${{targets.subpkgdir}}/etc/nginx/conf.d
+            install -m644 -D nginx.conf ${{targets.subpkgdir}}/etc/nginx/
+            install -m644 -D nginx.default.conf ${{targets.subpkgdir}}/etc/nginx/conf.d
+            # Follow Docker Image and link logs to to stdout/stderr
```

Comment:
Can we link to a source of truth that the Docker image does this? This seems like it would be helpful if we're debugging this a year from now.

Reply:
Argh, you've led me to another rabbit hole. This happens in the Debian (latest) image:

https://github.com/nginxinc/docker-nginx/blob/73a5acae6945b75b433cafd0c9318e4378e72cbb/mainline/debian/Dockerfile#L103-L104

It doesn't happen in the alpine image it seems, so I don't currently understand how it works.

Reply:
Done